Jenkinsfile

```diff
@@ -84,14 +94,5 @@ pipeline {
                     undefinedFails: false
                     ])
         }
-        success {
```

Comment:
@danistrebel - you might have to remove this from PR as for me, I have a portal were this works